Classes/GlobalStateExplorers/DatabaseBrowser/FLEXTableListViewController.m

```diff
@@ -67,64 +67,71 @@ - (void)viewDidLoad {
             return [tableName localizedCaseInsensitiveContainsString:filterText];
         }
     ];
-    
+
     self.tables.selectionHandler = ^(FLEXTableListViewController *host, NSString *tableName) {
-        NSArray *rows = [host.dbm queryAllDataInTable:tableName];
-        NSArray *columns = [host.dbm queryAllColumnsOfTable:tableName];
-        NSArray *rowIDs = [host.dbm queryRowIDsInTable:tableName];
-        UIViewController *resultsScreen = [FLEXTableContentViewController
-            columns:columns rows:rows rowIDs:rowIDs tableName:tableName database:host.dbm
-        ];
-        [host.navigationController pushViewController:resultsScreen animated:YES];
+        FLEXSQLResult *result = [host.dbm queryTable:tableName];
```

Comment:
`FLEXTableListViewController` supports more than one db. You cannot have `FLEXSQLResult` dependencies here.